docs/advisories/meltdown-spectre.md

```diff
@@ -0,0 +1,105 @@
+## Meltdown and Spectre Advisory
+
+| | |
+|-------------|--------|
+| NAME         	| Meltdown and Spectre Hardware Issues |
+| Description  	| Systems with microprocessors utilizing speculative execution and branch prediction may allow unauthorized disclosure of information to an attacker with local user access via a side-channel analysis. 	|
+| CVE(s)       	| [CVE-2017-5753](https://nvd.nist.gov/vuln/detail/CVE-2017-5753)  [CVE-2017-5754](https://nvd.nist.gov/vuln/detail/CVE-2017-5753) |
+| NVD Severity 	| medium (attack range: local) |
+| Last Updated  | Jan 01 2018 |
+
+##  Details
+
+This hardware exploit requires the installation of the Linux Kernel >= 4.4.110.
+kops is currently running the 4.4.x kernel.  A rolling-update or replacement of
+the kops ami image, with kops hosted on AWS is recommended. All platforms are
+affected not just AWS.
+
+Three CVEs have been released with spectre and meltdown.
+
+- Variant 1: bounds check bypass (CVE-2017-5753)
+- Variant 2: branch target injection (CVE-2017-5715)
+- Variant 3: rogue data cache load (CVE-2017-5754)
+
+Currently, Variant 1 and Variant 3 are solved with this advisory.
+
+
+### Impacted kops / kubernetes Components
+
+- kops maintained AMI
```

Comment:
How about:

"All linux kernels were vulnerable when running on affected hardware.  Fixed in 4.4.110 for 4.4, 4.9.75 for 4.9, 4.14.12 for 4.14."

And replace this whole list (we're going to move it above)